app/uk/gov/hmrc/agentsubscriptionfrontend/models/Address.scala

```diff
@@ -137,7 +136,4 @@ object Address{
 
     OFormat[Address](reads, formatAddressValue)
   }
-
-  val renderErrors: PostCodeError => String = (errors: PostCodeError) => errors.foldLeft("")(_+", "+_).substring(1)
```

Comment:
Does this have to be a `val`, can it be a `def` instead? `def` is less advanced -> easier to understand